cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition/help-script.html

```diff
@@ -23,3 +23,39 @@
     but if you specify multiple parameters they must all be named:
 </p>
 <pre>stage name: 'Build', concurrency: 1</pre>
+
+<h3>Environment Variables</h3>
+
+A set of environment variables are made available to all Jenkins Job types, including Workflow Jobs.
+The following is a general list of variables (by name) that are available to all Job types. See the notes
+below the list for Workflow specific details.
+
+<p>
+    <iframe src="${rootURL}/env-vars.html" width="100%"></iframe>
+</p>
+
+The following variables are only available inside <code>node</code>s (i.e. not available outside):
+<ul>
+    <li>EXECUTOR_NUMBER</li>
+    <li>NODE_NAME</li>
+    <li>NODE_LABELS</li>
+    <li>WORKSPACE</li>
+</ul>
+
+<h4>Using Environment Variables</h4>
+
+Environemnt variables are injected into scripts through a variable named "<strong>env</strong>". This variable,
+like any other variable, can be used in the general flow of the script, or in variable substitutions e.g. when
+constructing email content when using the <code>mail</code> step:
+
+<p>
+<pre>
```

Comment:
IIRC `p` and `pre` are both block-level elements, so I would suggest `</p><pre><code>`. (BTW if you switch to Jelly/Groovy you have to produce well-formed X(HT)ML.)